### ISSUE TITLE
Stuck with "Getting initial mod page" when clicking on Mods tab of RedModManager

### ISSUE THREAD

**Issue by xtraspecialj** [ToniMacaroni/RedManager#5]
As title says, I just see "Getting initial mod page" with an empty grey bar underneath.  Have let it sit there for a very long time (over 15 minutes) and nothing happens.   I've closed and re-launched it several times and even rebooted computer.  Still never comes up.
![RedModManager_Stuck](https://github.com/ToniMacaroni/RedManager/assets/12704686/f504b8b6-de89-47d4-a206-cdeed71d47ac)

**Comment by FORREST199228** [comment]
Did you fix the issue having the same one

**Comment by xtraspecialj** [comment]
> Did you fix the issue having the same one

No.  Don't know where the log files are to dig into it myself, either.  So I can only wait to hear from the author on a possible fix.

**Comment by ToniMacaroni** [comment]
There was an update to the api for sotf-mods.
Downloading the new version from release page (1.1.5) should fix it.

**Comment by joridiculous** [comment]
the problem with this "manger" is it newer remember the game folder and it want load the mod page until you enter (for n-th time) the game folder. A simple INI file for manager to read would solve this.
"RedLoader" will still run when starting the game.

**Comment by wFirbolg** [comment]
Any updates on this... having the same issue.

**Comment by Spiricore** [comment]
Running into the same issue here.

**Comment by Spiricore** [comment]
> Running into the same issue here.
Fixed with the latest version. Cheers for the quick update, and the whole app in general.
